### PR TITLE
Testing Updates

### PR DESCRIPTION
**Pull request by priyankjalu89**
Not ready for merge.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,35 +13,27 @@ REMOTEDIR=deb.qcode.co.uk
...
-	rm -rf package
-
-install: tcl-package local-install
+test: package
+	cd $(TEMP_PATH)/test && tclsh all.tcl
 
-local-install:
+install:  
 	mkdir -p /usr/lib/tcltk/$(NAME)$(VERSION)
 	cp package/*.tcl /usr/lib/tcltk/$(NAME)$(VERSION)/
```
Comment (PeterChaplin) [inline Makefile:35]: Will the package directory exists without running `tcl-package` first?

Comment (priyankjalu89) [reply Makefile:35]: No it will not exist.

**Comment by priyankjalu89** [comment]
Ready for review.

File: Makefile [hunk 2]
```diff
@@ -11,35 +13,27 @@ REMOTEDIR=deb.qcode.co.uk
...
 package: check-version
 	# Copy files to pristine temporary directory
+	rm -rf $(TEMP_PATH)	
+	mkdir $(TEMP_PATH)
 	rm -rf package
 	mkdir package
 	curl --fail -K ~/.curlrc_github -L -o v$(VERSION).tar.gz https://api.github.com/repos/qcode-software/qcode-tcl/tarball/v$(VERSION)
-	tar --strip-components=1 --exclude Makefile --exclude description-pak --exclude doc --exclude docs.tcl --exclude package.tcl --exclude test --exclude test_all.tcl -xzvf v$(VERSION).tar.gz -C package
-	./package.tcl tcl package ${NAME} ${VERSION}
+	tar --strip-components=1 --exclude Makefile --exclude description-pak --exclude doc --exclude docs.tcl --exclude package.tcl -xzvf v$(VERSION).tar.gz -C $(TEMP_PATH)
```
Comment (bvw) [inline Makefile:24]: keep lines below 80 chars long.

https://www.gnu.org/software/make/manual/html_node/Splitting-Lines.html

**Pull request merged**